hw2/05-form.html

```diff
@@ -10,10 +10,57 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
-    <main>
+  <body class="bg-primary container mt-5">
+    <main class=" bg-light rounded p-4 d-flex flex-column max-sm-75">
       <form>
-        <h1>Exercise 05 - Form</h1>
+        <div class="mb-3">
+          <label for="fullName" class="form-label">Full name</label>
+          <input type="text" name="full-name" class="form-control" id="fullName">
+        </div>
+        <div class="mb-3">
+          <label for="email-input" class="form-label">Email address</label>
+          <input type="email" name="email" class="form-control" id="email-input">
+        </div>
+        <div class="mb-3">
+          <label for="registration-status" class="form-label">Registration Status</label>
+          <select class="form-select" name="registration-status" id="registration-status" aria-label="Default select example">
+            <option selected>Choose an Option</option>
+            <option value="1">Registered</option>
+            <option value="2">Unregistered</option>
+            <option value="3">Waitlist</option>
+          </select>
+        </div>
+        <div class="form-group">
+          <legend>Have you taken any of the following courses?</legend>
+          <div class="form-check">
+            <input class="form-check-input" name="programmingLanguages" type="checkbox" value="option1" id="check1">
+            <label class="form-check-label" for="check1">
+              Programming Languages
+            </label>
+          </div>
+          <div class="form-check">
+            <input class="form-check-input" name="operatingSystems" type="checkbox" value="option2" id="check2">
+            <label class="form-check-label" for="check2">
+              Operating Systems
+            </label>
+          </div>
+          <div class="form-check">
+            <input class="form-check-input" name="fullstackWebdev" type="checkbox" value="option3" id="check3">
+            <label class="form-check-label" for="check3">
+              Full Stack Web Development
+            </label>
+          </div>
+        </div>
+        <div class="row mb-3">
+          <label class="form-check-label" for="text-input">Anything else?</label>
+          <textarea type="text" name="comments" id="text-input" class="form-check-label"></textarea>
```

Comment:
The spacing is a bit to close to the edge of the container for this textarea. You might want to add some spacing to the left.